minutes/tc39-outreach-2019-01-07.md

```diff
@@ -0,0 +1,112 @@
+    * MC: Can we potentially ship temporal as experimental in Node.js when it gets to Stage 3?
+    * GC: i think V8 would ship it, not node
+    * DE: Node should probably not in the shipping pipeline; it can be in npm and then in V8.
+    * BF: Node has some APIs which take numbers in milliseconds; if it's shipping in experimental, then we can apply usage of the new stdlib feature to Node Core, either in existing APIs or in a new APIs.
+    * DE: In those cases, would it make sense for Node to take a dependency on the npm module?
+    * QZ: It would be difficult to ship a new type in Node, given our strong compatibility guarantee. It'd be hard to go through the experimental process with this.
+    * BF: I agree, we should just create modules as examples of using host APIs with these new data types. We should probably figure out a process for, when new datatypes are added to JavaScript, how we want to prune things that are overlapping/similar. Luckily this is small for time in Node.
+* MD: is there a better channel to get feedback from Node, for example, broadcast about that from the Node.js twitter account?
+    * MB: We could tie into the CommComm user feedback initiative
+    * MD: We could explain to people what it is, how to use it, what it's for, etc, and we could collect the feedback and figure out next steps. We haven't found the best way to contact all the end users, but one thing we need is topics.
+    * DE: Maybe we could use this meeting to figure out how to set that feedback mechanism up.
+
+Concrete next steps:
+
+* MD: Bradley's suggestion is good; we need to find a volunteer (e.g., open an issue) to drive looking into this.
+* QZ: I think we can make a list, for specific proposals that may need some experimental examples that may need APIs built, we can make a list in the open standards repo, and try to get some help from there
+* MD: +1. Also, we can think about what different kinds of feedback--if we want to get more feedback from the Node.js collaborator base, we can open an issue at the right key point, or try to get it more broadly from end users (e.g., in the user feedback working group repo)
+* DE: Maybe we have a general framework here, 1. Experimental module 2. Gather feedback from collaborators 3. Gather feedback from the broader community.
+* MD: Experimenting would look different in different cases
+* BF: Here, there are existing date APIs, so that affects it. There's also the broad issue of, does this make it easier and harder to write programs, which we can do by examining existing ones. Node can make recommendations to champions about what to consider from Node Core, and leave it up to the champions for how to apply this advice.
+* MD: I think Joyee's issue about what's being requested is good, and then see if we can find people to volunteer to do that work.
+
+#### Built-in modules
+
+#### Top-level await
+
+#### Zones
+
```

Comment:
We are currently blocking this (on error handling)

Reply:
Probably for a good reason (after domains)

Reply:
What we decided in the meeting was that we should look into these topics in issues on this repository, and schedule meetings about the particular topic as necessary. Could you start an issue about Zones and describe the issue?

Reply:
looks like https://github.com/domenic/zones/issues/9 has relevant info

Reply:
This thread is really long, and the last comment I see is from 2016. Let's start a new issue to explain the current status and pick out the relevant things from history.

Reply:
@littledan did you started a new issue?